src/runtime/components/forms/SelectMenu.vue

```diff
@@ -481,7 +490,7 @@ export default defineComponent({
       }
 
       return props.options || []
-    }, [], {
+    }, props.options || [], {
```

Comment:
@benjamincanac Since the `selected` computed proeprty uses the `computedAsync` `options` to get the possible values, I initialized it to `props.options` so that it isn't empty during the initial render of the component to prevent a flash.
However, now that I'm thinking about it, shouldn't we just use `props.options` directly in the `selected` computed, instead of `options` - since we don't want to compare the model values with only the search results, but rather the whole options?

Reply:
I used `options` because that's what was used previously, however, unless I'm missing something, I think that `props.options` should have been used there as well:

<img width="886" alt="image" src="https://github.com/user-attachments/assets/4e917c06-3d0b-4384-8564-f011e7d02310">

Though this only affect things if someone uses asynchronous search.